lib/models/builder.js

```diff
@@ -27,9 +27,6 @@ class Builder extends CoreObject {
   constructor(options) {
     super(options);
 
-    this.broccoli2 = isExperimentEnabled('BROCCOLI_2');
-    this.systemTemp = isExperimentEnabled('SYSTEM_TEMP');
```

Comment:
out of curiosity, what's the reason for this change?

Reply:
It was something that I asked for in the original Broccoli 2 PR, but it didn't get done there and I didn't think it was blocking.

Basically, this makes removing the flag easier (just search for `isExperimentEnabled('BROCCOLI_2')` and remove all the conditionals), but you are totally correct that it isn't _required_...

Reply:
okay, makes sense, thanks